images/macos/macos-10.15-Readme.md

```diff
@@ -1,258 +1,231 @@
 ---
 title: GitHub Hosted Github Mojave 10.15 VM Image Updates
 description: Software used on build machines
-date: Week 2
+date: Week 3
 ---
 
-#### Xcode 11.2.1 set by default
+#### ⚠️ We are going to change default Xcode to 11.3.1 in two weeks.
```

Comment:
Shouldn't we still have a line about what Xcode is default now? so it will be visible from which version we a switching from